pkgs/top-level/all-packages.nix

```diff
@@ -17793,7 +17793,9 @@ with pkgs;
   coq_8_4 = coqPackages_8_4.coq;
   coq_8_5 = coqPackages_8_5.coq;
   coq_8_6 = coqPackages_8_6.coq;
-  coq_HEAD = callPackage ../applications/science/logic/coq/HEAD.nix {};
+  coq_HEAD = callPackage ../applications/science/logic/coq {
+    version = "8.7pre-2017-05-28";
```

Comment:
@rht Well, once the tag is put it will be `8.7beta1` not `8.7pre`, otherwise I wouldn't have used this here. Moreover, we won't be referring to this new version with variable name `coq_HEAD` but `coq_8_7` (see the history of this derivation).